cmd/cosign/cli/verify.go

```diff
@@ -126,6 +126,11 @@ against the transparency log.`,
 				Offline:                      o.CommonVerifyOptions.Offline,
 				TSACertChainPath:             o.CommonVerifyOptions.TSACertChainPath,
 				IgnoreTlog:                   o.CommonVerifyOptions.IgnoreTlog,
+				MaxWorkers:                   o.CommonVerifyOptions.MaxWorkers,
+			}
+
+			if o.CommonVerifyOptions.MaxWorkers == 0 {
+				v.MaxWorkers = 10
```

Comment:
let's return an error here that this is an invalid value so that people aren't surprised by what ends up happening

Reply:
done